docs/configuration.md

```diff
@@ -59,3 +59,9 @@ jobs: 8
 
 The number of jobs can be overridden by the `-j` or `--jobs` option in the CLI.
 See mutant's configuration file, [mutant.yml](/mutant.yml), for a complete example.
+
+#### `mutation_timeout`
+
+Specify the maximum time, in seconds, a mutation gets analysed.
+Past this time the mutation analysis gets terminated and the result is currently being
+reported as uncovered.
```

Comment:
I'll improve this via 2 iterations:

* Reporting timeouts separately in the env result.
* Allowing the user to set a global `timeout_success` to say: Timeouts are "intentional" here.